src/modules/launcher/Wox.Plugin/IPlugin.cs

```diff
@@ -17,5 +17,8 @@ public interface IPlugin
 
         // Localized description
         string Description { get; }
+
+        // Plugin ID for validating the plugin.json entry (It must be static for accessing it before loading the plugin.)
+        public static abstract string PluginID { get; }
```

Comment:
@crutkas , @davidegiacometti 
The new .Net 7 feature `static abstract` interface properties are not supported by the Moq framework. This causes the tests to fail. (https://github.com/moq/moq/issues/1398)

What should we do? There are three options I think:
1. Making it an interface member and throw `NotImplemented` when called as interface property. (Then you can compile plugins without having the property.)
2. Removing the affected test cases.
3. Splitting the interface into two nested interfaces and updating the test code to use the base interface (IPluginBase). *

*) Example
```
interface IPlugin : IPuginBase
{
  public static abstract  string PluginID { get; }
}

interface IPluginBase
{
  ...
  string Name { get; }
  string Description { get; }
}
```

Reply:
I decided to go with something like option 3.